CustomerOrganizer/CustomerOrganizer/MainWindow.py

```diff
@@ -1,7 +1,31 @@
 import tkinter
+from tkinter import *
+import FileIO
+from FileIO import *
 
-class Application(tkinter.Frame):
+class MainWindow(tkinter.Tk):
     def __init__(self, master = None):
         super().__init__(master)
         self.master = master
-        self.pack()
\ No newline at end of file
+        self.toolBar()
+
+        self.customer = FileIO.FileIO()
+
+    def donothing():
```

Comment:
I know this is a temporary name and it's going to change anyway, but let's practice consistency. Camel case this please.

Reply:
Actually I'm noticing all the tkinter methods use underscores. Maybe we should keep consistent with that instead. (So change this to do_nothing instead. Also please update the saveFile function to use save_file while you're at it.) We can totally discuss this if you think that's a bad idea.

Reply:
underscore ftw